go.mod

```diff
@@ -28,7 +28,6 @@ require (
 	golang.org/x/sys v0.11.0 // indirect
 	golang.org/x/term v0.11.0 // indirect
 	golang.org/x/text v0.12.0 // indirect
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0 // indirect
```

Comment:
keep this one. this is used in make install. Ignore any recommendation of code plugins to remove this. This is set so that everyone uses the same version to generate the proto files.

Reply:
Done